### DISCUSSION TITLE
3D-U-net for semantic segmentation of brain tissues

### DISCUSSION THREAD

**Discussion by AbdulrahmanMohammed2021** [TorchIO-project/torchio#501] [General]
Hi,
I'm trying to semantically segment brain tissues into skull, CSF, white matter and grey matter using 3D unet. I've already my own data [256 256 192]. 

Can you please a suggest or do you know a torchio code for that task?

Thanks

**Comment by fepegar** [comment]
Hi, @AbdulrahmanMohammed2021. I suggest you go through the tutorials: https://github.com/fepegar/torchio/tree/master/examples
